### ISSUE TITLE
Create core directories if they do not exist.

### ISSUE THREAD

**Issue by mohitranka** [dgraph-io/dgraph#130]
Currently, if the directory does not exist for mutation it throws an error, and quits.

```
./dgraph --mutations /dgraph/m --postings /dgraph/p --uids /dgraph/u  
2016/07/12 15:55:34 num_cpu: 4. prev_maxprocs: 4. Set max procs to num cpus  
2016/07/12 15:55:34 Unable to find dir: stat /dgraph/m: no such file or directory  
```
Dgraph can instead create the directories and give a slightly better experience. 

Thoughts?

**Comment by mohitranka** [comment]
Case in point: #136

**Comment by manishrjain** [comment]
Makes sense, should be relatively trivial to add. Do you want to send us a PR, @mohitranka?

**Comment by mohitranka** [comment]
Merged to develop with commit 9d919395a698ed660af73ee9417f96630ed31c7e
